src/fraction/mod.rs

```diff
@@ -1,6 +1,7 @@
 pub use self::generic_fraction::GenericFraction;
 pub use self::sign::Sign;
 
+pub mod approx;
```

Comment:
For sure. Do you think it would make sense to just keep `approx` behind `with-bigint`, or should there be a separate feature entirely for `approx`? The latter seems reasonable to me on the grounds that this crate is primarily focused around lossless maths (ergo many users probably won't care about `approx`) but then again it's an extra feature flag.

Reply:
I feel like a separate feature would be fine, as long as it depends on `with-bigint` and that dependency is documented. The vice-versa, people who only need a bigint and not approx would be able to build the lib without dead code and get a smaller size.